pkg/kube/client.go

```diff
@@ -573,9 +573,10 @@ func (c *client) PodExecCommands(podName, podNamespace, container string, comman
 			if len(stderr) > 0 {
 				err = fmt.Errorf("error exec'ing into %s/%s %s container: %v\n%s",
 					podName, podNamespace, container, err, stderr)
+			} else {
+				err = fmt.Errorf("error exec'ing into %s/%s %s container: %v",
+					podName, podNamespace, container, err)
```

Comment:
Not because this PR, but it feels wrong to me to log as `podname/podNamespace`. Since you are here, could you swap these two log args?